config/sonic-nas.sources.list

```diff
@@ -1,2 +1,3 @@
 deb https://dell-networking.bintray.com/sonic-apt/ jessie main
+deb http://10.14.1.29/sonic unstable main
```

Comment:
Of course - I will do that tonight :-)

Cliff Wichmann

Dell Networking
5450 Great America, San Jose, CA
650-336-5886




On Fri, Jul 29, 2016 at 4:31 PM -0700, "lguohan" <notifications@github.com<mailto:notifications@github.com>> wrote:


In config/sonic-nas.sources.list<https://github.com/Azure/sonic-build-tools/pull/8#discussion_r72874610>:

> @@ -1,2 +1,3 @@
>  deb https://dell-networking.bintray.com/sonic-apt/ jessie main
> +deb http://10.14.1.29/sonic unstable main


hi cliff, there are many conflict in the pull request, can you update the scripts based the current master and update this commit?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Azure/sonic-build-tools/pull/8/files/cfdada90dff659273cd1c41c34fae455ded9a581#r72874610>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AQVlyqneaaAR_zeepmtXb_W0lu2E2gAsks5qao1agaJpZM4JW8t3>.